dspace-xmlui/src/main/webapp/i18n/messages.xml

```diff
@@ -2504,4 +2511,66 @@
+    <message key="xmlui.scidir.live-import-selected.import-action-workspace">Send imported items to workspace</message>
+    <message key="xmlui.scidir.live-import-selected.import-action-workflow">Send imported items to workflow</message>
+    <message key="xmlui.scidir.live-import-selected.import-action-archive">Archive imported items</message>
+    <message key="xmlui.scidir.live-import-selected.submit-cancel">Cancel</message>
+    <message key="xmlui.scidir.live-import-selected.submit-import">Import</message>
+    <message key="xmlui.scidir.live-import-selected.collection">Select collection:</message>
+    <message key="xmlui.scidir.live-import-selected.action">Select action:</message>
+
+    <message key="xmlui.scidir.live-import-action.success">Import finished successful</message>
+    <message key="xmlui.scidir.live-import-action.failure">Import failed</message>
+
+    <message key="xmlui.Submission.submit.LiveImportStep.submit_lookup">Search</message>
+    <message key="xmlui.Submission.submit.LiveImportStep.submit_skip">Skip Import</message>
+    <message key="xmlui.Submission.submit.LiveImportStep.title">Search</message>
+    <message key="xmlui.Submission.submit.LiveImportStep.lookup_help"><p>Fill in at least one of the following fields and press "Search". A list of all matching publications will be shown. Import a publication by clicking "import".</p><p>Importing a publication will detele all metadata that was already added to this item!</p></message>
+    <message key="xmlui.Submission.submit.LiveImportStep.lookup_help2">Use the "Search" button to search publications in ScienceDirect. Continue without importing using the "Next" button.</message>
+    <message key="xmlui.Submission.submit.LiveImportStep.submit_import_item">Imported Publication:</message>
+    <message key="xmlui.Submission.submit.LiveImportStep.records-found">{0} records found, now showing {1} to {2}:</message>
+    <message key="xmlui.Submission.submit.LiveImportStep.no-records">No records found</message>
+    <message key="xmlui.Submission.submit.LiveImportStep.submit_results">Search results</message>
+    <message key="xmlui.Submission.submit.progressbar.liveimport">ScienceDirect Import</message>
+
+
+    <message key="com.atmire.sciencedirect.embed.ElsevierEmbed.title">ScienceDirect embed page</message>
+    <message key="com.atmire.sciencedirect.embed.ElsevierEmbed.trail">ScienceDirect embed page</message>
+    <message key="com.atmire.sciencedirect.embed.ElsevierEmbed.title_error">The embed page cannot be rendered</message>
+    <message key="com.atmire.sciencedirect.embed.ElsevierEmbed.error.embed_page_disabled">The embed pages have been disabled</message>
+    <message key="com.atmire.sciencedirect.embed.ElsevierEmbed.error.not_an_item">There is no item associated with this URL</message>
+    <message key="com.atmire.sciencedirect.embed.ElsevierEmbed.error.invalid_pii">The given PII does not match with the item</message>
```

Comment:
Just realized all these message keys wrong say `com.atmire`. Not a big deal, but something to fix.